exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,6 @@
-const name = " Daniel  ";
+const myName = " Daniel  ";
+let nameLength = myName.trim();
```

Comment:
Hi Amalia, thanks for the review. This test passed, I'm not sure I understand your comment.

Reply:
Hi Ika, this is not about whether the test passed or not. This is about the consistency and correctness of the code, which cannot be verified by the test. ``nameLength`` is not the correct name for this since it's actually trimming.

Reply:
Sorry I forgot to push my code after making corrections. Can you please try again, thanks